utils/build-script-impl

```diff
@@ -2332,6 +2344,24 @@ for host in "${ALL_HOSTS[@]}"; do
                             -DLLDB_ALLOW_STATIC_BINDINGS=1
                         )
                         ;;
+                     haiku-*)
```

Comment:
Nit: indentation is inconsistent as compared to `cygwin-*)` above and `macosx-*)` below.